File: src/main/java/me/diax/diax/commands/fun/Say.java
package me.diax.diax.commands.fun;

import me.diax.comportment.jdacommand.Command;
import me.diax.comportment.jdacommand.CommandAttribute;
import me.diax.comportment.jdacommand.CommandDescription;
import me.diax.diax.util.Emote;
import net.dv8tion.jda.core.entities.Message;

@CommandDescription(
        name = "say",
        triggers = {
                "say"
        },
        description = "[message]",
        attributes = {
                @CommandAttribute(key = "private"),
        }
)
public class Say implements Command {

    @Override
    public void execute(Message message, String s) {
        if (s.isEmpty()) {
            message.getChannel().sendMessage(Emote.X + " - Specify something to say.").queue();
        } else {
            message.getChannel().sendMessage(s).queue();
        }
    }
}

File: src/main/kotlin/me/diax/diax/commands/fun/Say.kt
package me.diax.diax.commands.`fun`

import me.diax.comportment.jdacommand.Command
import me.diax.comportment.jdacommand.CommandAttribute
import me.diax.comportment.jdacommand.CommandDescription
import me.diax.diax.util.Emote.X
import net.dv8tion.jda.core.entities.Message

@CommandDescription(name = "say", triggers = arrayOf("say"), description = "[message]",
        attributes = arrayOf(CommandAttribute(key = "category", value = "fun")))
class Say : Command {

    override fun execute(message: Message, s: String) {
        if (s.isEmpty()) {
            message.channel.sendMessage("$X - Specify something to say.").queue()
        } else {
            message.channel.sendMessage(s).queue()
        }
    }
}